develop/dev-guide-sample-application-aws-lambda.md

```diff
@@ -0,0 +1,216 @@
+
+3. If you have not set a password yet, click **Create password** to generate a random password.
+
+   <Tip>If you have generated a password before, you can use the original password directly, or click **Reset password** to generate a new password.</Tip>
+
+4. Copy and paste the corresponding connection string to `env.json`. The following is an example:
+
+   ```json
+   {
+     "Parameters": {
+       "TIDB_HOST": "{gateway-region}.aws.tidbcloud.com",
+       "TIDB_PORT": "4000",
+       "TIDB_USER": "{prefix}.root",
+       "TIDB_PASSWORD": "{password}"
+     }
+   }
+   ```
+
+   Replace the placeholders in `{}` with the values obtained in the connection dialog.
+
+5. [Copy and configure the corresponding connection string](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html) in Lambda Function.
+
+  ![quickstart-lambda-env](/media/develop/quickstart-lambda-env.png)
+
+</div>
+
+<div label="TiDB Self-Hosted">
+
+1. Copy and paste the corresponding connection string to `env.json`. The following is an example:
+
+   ```json
+   {
+     "Parameters": {
+       "TIDB_HOST": "{tidb_server_host}",
+       "TIDB_PORT": "4000",
+       "TIDB_USER": "root",
+       "TIDB_PASSWORD": "{password}"
+     }
+   }
+   ```
+
+   Replace the placeholders in `{}` with the values obtained in the **Connect** window.
+
+2. [Copy and configure the corresponding connection string](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html) in Lambda Function.
+
+  ![quickstart-lambda-env](/media/develop/quickstart-lambda-env.png)
+
+</div>
+
+</SimpleTab>
+
+### Step 4: Run the sample application locally
+
+1. Install the [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/install-sam-cli.html).
+
+2. Run the following command to invoke the sample Lambda function:
+
+    ```bash
+    sam local invoke --env-vars env.json -e event.json "tidbHelloWorldFunction"
+    ```
+
+3. Check the output in the terminal. If the output is similar to the following, the connection is successful:
+
+    ```bash
+    {"statusCode":200,"body":"{\"results\":[{\"Hello World\":\"Hello World\"}]}"}
+    ```
+
+## Example codes
+
+Complete code snippets are available in the [tidb-aws-lambda-quickstart](https://github.com/tidb-samples/tidb-aws-lambda-quickstart) GitHub repository.
+
+You can refer to the following key code snippets to complete your application development.
+
+### Connect to TiDB
+
+```typescript
+import mysql from 'mysql2';
+
+const pool = mysql.createPool({
+  host, // TiDB host, for example: {gateway-region}.aws.tidbcloud.com
+  port, // TiDB port, default: 4000
+  user, // TiDB user, for example: {prefix}.root
+  password, // TiDB password
+  database, // TiDB database name, default: test
+  ssl: {
+    minVersion: 'TLSv1.2',
+    rejectUnauthorized: true,
+  },
+  connectionLimit: 1,
```

Comment:
Maybe we should briefly explain the reason why the `connectionLimit`,`maxIdle` is set to `1`.

- https://www.cockroachlabs.com/docs/v23.1/serverless-function-best-practices#use-connection-pools
- https://www.prisma.io/docs/guides/deployment/serverless/deploy-to-aws-lambda#connection-pooling
- https://www.prisma.io/docs/guides/performance-and-optimization/connection-management#without-an-external-connection-pooler